apps/coscrad-frontend/src/app/app.tsx

```diff
@@ -23,10 +29,69 @@ export function App() {
 
         const aggregateTypeFromMessage = message.aggregateCompositeIdentifier?.type;
 
-        if (isAggregateType(aggregateTypeFromMessage))
+        // TODO update this
```

Comment:
Let's be sure neither notification service is providing aggregate IDs (but only types) at this point.

Reply:
Done